### PR TITLE
Created footer component

### PR DESCRIPTION
**Pull request by Othmanosx**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import logo from './logo.svg';
 import './App.css';
+import Navbar from './components/Navbar/Navbar.jsx'
```
Comment (maqalaqil) [inline src/App.js]: remove anything from app.js

**Review by maqalaqil — APPROVED** [review]
change the app.js

**Review by Ravyar19 — APPROVED** [review]

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+      label: "My Account",
+      href: "/account"
+    },
+    {
+      label: "Log Out",
+      href: "/logout"
+    }
+  ];
+  
+  const useStyles = makeStyles(() => ({
```
Comment (murtaja1) [inline src/components/Navbar/Navbar.jsx:47]: create a style file for all the styling put useStyles in it.

**Review by murtaja1 — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (Othmanosx) [reply src/App.js]: fixed app.js, check 8c1b4dc11be732d7eb072979898ccfd5fa3fd607

**Pull request merged**